AirQualityMonitoring/schema.json

```diff
@@ -0,0 +1,166 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "$schemaVersion": "0.0.1",
+    "$id": "https://smart-data-models.github.io/dataModel.Environment /EnvAQM/schema.json",
+    "title": " Smart Data Models - EnvAQM",
+    "description": "Air Quality Monitoring (AQM) Data Model.",
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/GSMA-Commons"
+        },
+        {
+            "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons"
+        },
+        {
+            "properties": {
+                "deviceInfo": {
+                    "type": "object",
+                    "description": "Property. Information about the device associated with the observations.",
+                    "properties": {}
+                },
+                "observationDateTime": {
+                    "type": "string",
+                    "format": "date-time",
+                    "description": "Property. Model:'https://schema.org/DateTime'. Last reported time of observation."
+                },
+                "deviceStatus": {
+                    "type": "string",
+                    "description": "Property. Model:'https://schema.org/Text'. Indicates the status of physical device or devices.\n"
+                },
+                "atmosphericPressure": {
+                    "type": "number",
+                    "description": "Property. Model:'https://schema.org/Number'. Observed air (atmospheric or barometric) pressure."
+                },
+                "airQualityIndex": {
+                    "type": "number",
+                    "description": "Property. Model:'https://schema.org/Number'. Overall Air Quality Index (AQI) for the observed air quality."
+                },
+                "airQualityLevel": {
+                    "type": "string",
+                    "description": "Property. Model:'https://schema.org/Text'. Air Quality Category Indication. Qualitative level defined according to the local health agencies. For example, 'GOOD', 'MODERATE', 'POOR', 'UNHEALTHY', 'SEVERE', 'HAZARDOUS' etc."
+                },
+                "aqiMajorPollutant": {
+                    "type": "string",
+                    "description": "Property. Model:'https://schema.org/Text'. Major pollutant in the Air Quality Index (AQI)."
+                },
+                "": {
+                    "type": "object",
+                    "description": "Property. ",
+                    "properties": {}
```

Comment:
Object's properties should be defined. there are approaches for generic needs. I'd mail you if necessary.